itn-installer.sh

```diff
@@ -50,7 +50,7 @@ chmod +x /opt/dusk/bin/*
 #mkdir -p /opt/dusk/installer/rusk
 #tar xf /opt/dusk/installer/rusk.tar.gz --directory /opt/dusk/installer/rusk
 #mv /opt/dusk/installer/rusk/rusk /opt/dusk/bin/
-chmod +x /opt/dusk/bin/rusk
+#chmod +x /opt/dusk/bin/rusk
```

Comment:
This is only needed if you uncomment previous lines, else the line 46 is already taking care of setting the proper exec right.